relay-quotas/src/redis.rs

```diff
@@ -695,7 +695,7 @@ mod tests {
 
         let rate_limiter = build_rate_limiter();
         let mut client = rate_limiter.pool.client().expect("get client");
-        let mut conn = client.connection();
+        let mut conn = client.connection().expect("Redis connection");
```

Comment:
Should we set a more meaningful error message? For example:
```suggestion
        let mut conn = client.connection().expect("Establish Redis connection");
```